client/my-sites/customer-home/cards/notices/celebrate-site-launch-v2/index.jsx

```diff
@@ -14,6 +14,7 @@ import getSiteTaskList from 'state/selectors/get-site-task-list';
 import isSiteChecklistComplete from 'state/selectors/is-site-checklist-complete';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import CelebrateNotice from '../celebrate-notice-v2';
+import { NOTICE_CELEBRATE_SITE_LAUNCH } from '../../constants';
```

Comment:
Also nonblocking: I tend to prefer absolute paths for easier searching/refactors. Say `'my-sites/customer-home/cards/constants.js'` vs `'../../constants'`